supply-chain-security/compromises/compromise-definitions.md

```diff
@@ -0,0 +1,52 @@
+# Types of Supply Chain Compromise
+
+This document serves to provide a consistent set of definitions for the Supply Chain Compromise Catalog.  The types of compromise may be expanded in the future as new or more distinct compromises occur.  When this happens, these definitions will be updated.
+
+Index:
+
+
+* [Dev Tooling](#dev-tooling)
+* [Negligence](#negligence)
+* [Publishing Infrastructure](#publishing-infrastructure)
+* [Source Code Compromise](#source-code-compromise)
+* [Trust and Signing](#trust-and-signing)
+
+## Source Code Compromise
+Occurs when a source code repository (public or private) is manipulated intentionally by the developer or through a developer or repository credential compromise.  Source Code compromised can also occur when source code in open-source dependencies is manipulated in a manner to make them more useful (more appealing for use in order to guarantee download of malicious content, trojan horse like).
```

Comment:
Ah okay - i understand what you mean by this now. Would "Source code compromise can also occur with intentional introduction of security backdoors and bugs in Open Source code contributions by malicious actors"

Reply:
changed.